docs/index.md

```diff
@@ -53,7 +53,7 @@ for hosting your (binary) artifacts, regardless or your repository layout.
     Search providers offer a way to execute searches against different search engines. By default, searches are executed 
     against OrientDB, unless a search provider has been specified. [Click here for more details](./developer-guide/search-providers.md)
 
-    * [x] [Artifact Query Language (AQL) to make searching easy and consistent between layouts](./user-guide/artifact-query-language.md) 
+    * [x] [Artifact Query Language (AQL)] to make searching easy and consistent between layouts(./user-guide/artifact-query-language.md) 
```

Comment:
This link is broken :)

Reply:
The link was working for me. I changed the linked text to only include Artifact Query Language (AQL) instead of the entire sentence. I will recheck the link when I get back home.